app/models/api_model.php

```diff
@@ -79,7 +102,7 @@
         }
 
         $supported = array_unique(array_values($project->getLocalesMapping($request->query['store'])));
-        $json = array_values(array_intersect($done, $supported));
+        $json = empty($json) ? array_values(array_intersect($done, $supported)) : $json;
```

Comment:
Where is $json defined? The only place where I see it defined is in a very edge case line 83, which means that in normal cases $json is not set, I am not sure what the result of empty on an unset value is.